docs/bucket/replication/README.md

```diff
@@ -129,6 +129,8 @@ Replication status can be seen in the metadata on the source and destination obj
 
 To perform bi-directional replication, repeat the above process on the target site - this time setting the source bucket as the replication target.
 
+It is recommended that replication be run in a system with atleast two CPU's available to the process, so that replication can run in its own thread.
```

Comment:
```suggestion
It is recommended that replication be run in a system with at least two CPU's available to the process, so that replication can run in its own thread.
```